app/src/services/util/track.js

```diff
@@ -0,0 +1,5 @@
+import ReactGA from 'react-ga';
+
+export const trackPage = () => {
```

Comment:
Shall we refactor the function name?

something like `trackPageWithGoogleAnalytics`

Reply:
Lets change the file name to `googleAnalytics.js`

Reply:
Okay. Updated